src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommand.java

```diff
@@ -7,9 +7,57 @@
     private String requestId;
     private Command command;
     private int timeout;
-    private  int requestLength;
+    private int requestLength;
     private String operationId;
     private int responseLength;
     private int execTime;
 
+    public ExecutedCommand(String parse)
+    {
+    	String[] parseArray = parse.split("  ");
+    	this.requestLength = 0;
+    	this.responseLength = 0;
+    	this.timeout = 0;
+    	this.execTime = 0;
+    	for (int i=0; i<parseArray.length-1; i++)
+    	{
+    		if (parseArray[i].compareTo("requestId") == 0)
+    			this.requestId = parseArray[i+1];
+    		else if (parseArray[i].compareTo("commandId") == 0)
+    			this.command = new Command(Integer.parseInt(parseArray[i+1]),parseArray[i+3]);
```

Comment:
I think commandId is always int. comandName and comandId go together in logfile, after comandId(parseArray[i]), we see comandName(i+2), and i+3 was value of the comandName